src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const stateClone = { ...state };
```

Comment:
Consider renaming 'stateClone' to 'stateCopy' to better convey the meaning that it is a copy of the current state.